### DISCUSSION TITLE
Overriding input directory for submodules

### DISCUSSION THREAD

**Discussion by kolboch** [detekt/detekt#3375] [Q&A] [answered]
Working with big project and have a lot of not so standard directories I wonder if it's possible to override input directory per module while using detekt {} closure and allprojects { include "[...] detekt.gradle } to run detekt across the projects. It would greatly improve readability and probably will let me finally get some reliable baseline.xml... Is it possible and I'm missing that somehow in the docs? Should I maybe use separate detekt tasks and try to run them together from the main build.gradle file?

**Comment by BraisGabin** [comment, marked as answer]
Sorry for the late response. And I'm not sure if I understood you correctly. But you can configure detekt in a allproject and then override the configuration you want in the build.gradle of the modules you need.
